source/renderer/app/api/api.js

```diff
@@ -2851,16 +2851,18 @@ const _createAssetFromServerData = action(
       fingerprint,
       metadata,
     } = data;
+    const uniqueId = `${policyId}${assetName}`;
```

Comment:
It's good that this is using the encoded `assetName` and not attempting to decode as ASCII, etc.